readme.md

```diff
@@ -218,6 +218,7 @@
 - [Preact](https://github.com/ooade/awesome-preact) - App framework.
 - [Progressive Enhancement](https://github.com/jbmoelker/progressive-enhancement-resources)
 - [Next.js](https://github.com/unicodeveloper/awesome-nextjs) - Framework for server-rendered React apps.
+- [Styled-Components](https://github.com/styled-components/awesome-styled-components) - Visual primitives for the component age.
```

Comment:
This description doesn't really explain very well what it is. It's more like marketing talk.